tests/test_trainer.py

```diff
@@ -3,6 +3,8 @@
 """
 import sys
 sys.path.append('src/development')
+sys.path.append('src/runtime')
```

Comment:
I don't think this is really required though

Reply:
somehow when I try in clean environment it raise error

Reply:
using append could cause an issue if we already have another vortex path in the PYTHONPATH previously, as append will only put this path to the last path, so it would not really have any effect.  
I would prefer to use `.insert(0, path)` just to be safe for such situation.

Reply:
what is the error?

Reply:
src/development/vortex/development/core/factory.py:18: in <module>
    from vortex.runtime.factory import create_runtime_model
E   ModuleNotFoundError: No module named 'vortex.runtime'

Reply:
i think because currently this append mechanism is spread out to our test case, we can improve this in another issue